crates/pipeline-manager/src/integration_test.rs

```diff
@@ -1300,16 +1300,16 @@ async fn parse_datetime() {
     let response = config
         .post_json(
             "/v0/pipelines/test/ingress/t1?format=json&update_format=raw",
-            r#"{"t":"13:22:00","ts": "2021-05-20 12:12:33","d": "2021-05-20"}
-            {"t":" 11:12:33.483221092 ","ts": " 2024-02-25 12:12:33 ","d": " 2024-02-25 "}"#
+            r#"{"t":"13:22:00","ts": "2021-05-20T12:12:33Z","d": "2021-05-20"}
```

Comment:
I assume these changes are unrelated to variant

Reply:
yes, this is related to the new default encoding for timestamps